dietpi/dietpi-drive_manager

```diff
@@ -178,7 +176,7 @@ _EOF_
 		local cmd_scrape_string=''
 		local index=0
 		G_DIETPI-NOTIFY 0 'Detecting currently mounted drives, please wait...'
-		df -Ph | tail -n +2 | sed '/tmpfs/d' | sed '/^udev/d' > /tmp/.dietpi-drive_manager_df_tmp
+		df -Ph | tail -n +2 | sed '/^tmpfs/d' | sed '/^udev/d' > /tmp/.dietpi-drive_manager_df_tmp
```

Comment:
Reason for non^, on RPi:
```
devtmpfs        460M     0  460M   0% /dev
```

Reply:
@Fourdee 
Ah okay, on VM `df -PhT` (+fs type) showed devtmpfs as fs type, but not name. Makes sense then 👍.